charts/camunda-platform-8.8/templates/web-modeler/_helpers.tpl

```diff
@@ -191,14 +191,8 @@ Define match labels for Web Modeler websockets to be used in matchLabels selecto
         (include "webModeler.postgresql.fullname" .)
         (.Values.webModelerPostgresql.auth.database)
       -}}
-  {{- else if .Values.webModeler.restapi.externalDatabase.url -}}
+  {{- else -}}
     {{- .Values.webModeler.restapi.externalDatabase.url -}}
-  {{- else if .Values.webModeler.restapi.externalDatabase.host -}}
-    {{- printf "jdbc:postgresql://%s:%s/%s"
-        .Values.webModeler.restapi.externalDatabase.host
-        (toString (.Values.webModeler.restapi.externalDatabase.port))
-        (.Values.webModeler.restapi.externalDatabase.database)
-      -}}
```

Comment:
This looks like we're dropping support for `externalDatabase.host`, `externalDatabase.port`, and so on.

We normally do not just drop support for options like this. we typically talk with product management about a timeframe schedule notifying users of the setting to be removed, and a schedule for removal.